project_resstock_testing/housing_characteristics/Location EPW.tsv

```diff
@@ -1,2 +1,217 @@
-Option=USA_AZ_Phoenix-Sky.Harbor.Intl.AP.722780_TMY3.epw	Option=USA_CA_Los.Angeles.Intl.AP.722950_TMY3.epw	Option=USA_GA_Atlanta-Hartsfield-Jackson.Intl.AP.722190_TMY3.epw	Option=USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw	Option=USA_TX_Houston-Bush.Intercontinental.AP.722430_TMY3.epw	Option=USA_WA_Seattle-Tacoma.Intl.AP.727930_TMY3.epw
-0.166666667	0.166666667	0.166666667	0.166666667	0.166666667	0.166666667
```

Comment:
I wonder if we should consider a different format for these super sparse matrices. I think this works for now, but other options might be to have the "TMY3" option point to the S3 url and just avoid needing to assign a EPW file??

Reply:
Not sure I follow.

Reply:
Sorry I meant to comment on `project_resstock_national/housing_characteristics/Location EPW.tsv` I made a note to discuss later: https://trello.com/c/fekESvE3